src/test/java/de/uhd/ifi/se/decision/management/jira/extraction/gitclient/TestSetUpGit.java

```diff
@@ -31,6 +31,100 @@ public static void setUpBeforeClass() throws IOException {
 		makeExampleCommit("readMe.txt", "TODO Write ReadMe", "Init Commit");
 		makeExampleCommit("readMe.txt", "Self-explanatory, ReadMe not necessary.", "TEST-12: Explain how the great software works");
 		makeExampleCommit("GodClass.java", "public class GodClass {}", "TEST-12: Develop great software");
+
+		/*
+		* following mockcommits are for TangledCommitDedetection
+		* 1. and 2. commit are not tangled commits
+		* 3. and 4. commits are tangled commits, but the package distance of 4. commit is greater than 3
+		* excepted order on codeSummarize: 4->3->2->1
+		* */
+
+		makeExampleCommit("untangled.java", "package de.uhd.ifi.se.decision.management.jira.extraction.impl;\n" +
+				"\n" +
+				"public class Main {\n" +
+				"    public static void  main(String[] args) {\n" +
+				"        System.out.println(\"Hello World!\");\n" +
+				"    }\n" +
+				"}\n", "Test-12 add main");
+		makeExampleCommit("untangled2.java", "package de.uhd.ifi.se.decision.management.jira.extraction.impl;\n" +
```

Comment:
Wouldn't it make more sense to create files for these test contents and just read them out from these files?
The source code readability suffers from current approach...